measurements/api/measurements.py

```diff
@@ -140,10 +139,12 @@ def api_list_report_files():
     }
     results = []
     for row in q:
-        bucket_date, filename = row.textname.split('/')
-        url = get_download_url(current_app, bucket_date, filename)
+        download_url = urljoin(
+            current_app.config['BASE_URL'],
+            '/files/download/%s' % row.textname
```

Comment:
What is `textname`? The textual name / hash / id of a report?

Reply:
`textname` is textual name of a report file.
`filename` refers to blob storing that file: *.tar.lz4 or *.json.lz4

Reply:
Ah, I see: it's the name of the file inside the blog. Thanks.